src/main/java/com/example/petbuddybackend/entity/address/Address.java

```diff
@@ -0,0 +1,33 @@
+package com.example.petbuddybackend.entity.address;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class Address {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false, length = 50)
+    private String city;
+
+    @Column(nullable = false, length = 15)
+    private String zipCode;
+
+    @Column(nullable = false, length = 50)
+    private Voivodeship voivodeship;
+
+    @Column(nullable = false, length = 150)
+    private String street;
```

Comment:
chyba sa przypadki, ze budynek nie ma przypisanej ulicy

Reply:
Wiem że wsie mają adres nazwa_wsi numer. Ja to tak zinterpretowałem że nazwa_wsi jest też ulicą